<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>409</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CRUD Application</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: &quot;silver&quot;;
}
QWidget {
        background-color: &quot;gainsboro&quot;;
}

/* --------------------------------------- Style the BUTTONS --------------------------------------- */
QPushButton#btn_create, QPushButton#btn_update, QPushButton#btn_delete {
    color: &quot;black&quot;; /* Button Text */
    background-color: &quot;powderblue&quot;;
    padding: 10px;
    border-radius: 5px;
    border-width: 1px;
    border-color: &quot;lightslategrey&quot;;
    border-style: solid;
}
QPushButton::hover#btn_create, QPushButton::hover#btn_update, QPushButton::hover#btn_delete {
    color: &quot;white&quot;; /* Button Text */
    background-color: &quot;lightslategrey&quot;;
    border-color: &quot;powderblue&quot;;
    font-weight:bold;
}

QPushButton#btn_exit {
    color: &quot;black&quot;; /* Button Text */
    background-color: &quot;orange&quot;;
    padding: 8px;
    border-radius: 5px;
    border-width: 1px;
    border-color: &quot;red&quot;;
    border-style: solid;
    width:100px;
    height:10px;
}
QPushButton::hover#btn_exit {
    color: &quot;white&quot;; /* Button Text */
    background-color: &quot;red&quot;;
    border-color: &quot;orange&quot;;
    border-width: 0px;
    font-weight:bold;
}
/* --------------------------------------- TABLE  -----------------------------------*/
QTableView {
    gridline-color: &quot;orange&quot;;
    background-color: &quot;blanchedalmond&quot;;
    border-color: &quot;darkgrey&quot;;
    padding: 1px;
}
/* Header style */
QHeaderView::section {
    padding-bottom: 5px;
    padding-top: 5px;
    background-color: &quot;cadetblue&quot;;
    color: white;
    height: 25px;
    /* width: 150px; */
    font: 14px;
}
QTableView::item:selected {
    /* row selected  Green */
    background-color: &quot;darkseagreen&quot;;
}
QScrollBar {
    background-color: &quot;powderblue&quot;;
    width: 16px;
    border: none;
}
QScrollBar::handle {
    background-color: &quot;lightslategrey&quot;;
    margin: 16px 1px 16px 1px;
    border-radius: 6px;
}
/* --------------------------------------- Status Bar  -----------------------------------*/
QStatusBar {
        background: &quot;silver&quot;;
        border-top: 2px solid &quot;gray&quot;;
}
QStatusBar, QStatusBar QLabel {
        color: &quot;black&quot;;
}
QSizeGrip {
    image: url(window_grip.svg);
    width: 20px;
    height: 20px;
    background-color: &quot;silver&quot;;
}

/* --------------------------------------- Menu Bar  -----------------------------------*/
QMenuBar {
	background-color: &quot;silver&quot;;  /* - Top strip Bar  -*/
border-bottom: 2px solid &quot;gray&quot;;
}
QMenuBar::item:selected {
	background: &quot;lightslategrey&quot;;
	color: &quot;white&quot;;
	font-weight:&quot;bold&quot;;
}
QMenuBar QMenu {
    background-color: &quot;silver&quot;; /* - Drop down menu  -*/
}
QMenuBar QMenu::item:selected {
background-color: &quot;slategrey&quot;;
color: &quot;white&quot;;
}
 /* - 
  -*/



</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_title">
      <property name="text">
       <string>Device Database Application</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tv_tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_create">
        <property name="text">
         <string>Create</string>
        </property>
        <property name="icon">
         <iconset theme="contact-new"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_update">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset theme="edit-redo"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete">
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="icon">
         <iconset theme="user-trash"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="icon">
         <iconset theme="system-log-out"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuDevice">
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Device</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuApp">
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>App</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDevice"/>
   <addaction name="menuApp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
  </widget>
  <action name="actionCreate">
   <property name="icon">
    <iconset theme="list-add"/>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
   <property name="statusTip">
    <string>Create a new device.</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update the selected device.</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset theme="user-trash"/>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="statusTip">
    <string>Delete the selected device.</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="system-log-out"/>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit the application.</string>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
